addon/styles/_pix-icon-button.scss

```diff
@@ -16,31 +16,50 @@
     cursor: default;
   }
 
-  &:hover,
-  &:focus {
-    transition: background-color 0.2s ease;
-    background-color: $pix-neutral-15;
-    &:disabled {
-      background-color: transparent;
+  &:not(.pix-button--disabled) {
```

Comment:
Je ne comprends pas à quel moment cette classe intervient dans le composant `PixIconButton` ?
Elle n'est jamais initialisée, donc cette condition `not` est toujours vraie

Reply:
Tiens ça semble être un relicat de `<PixButton>`. La classe est ajoutée dans `pix-button-base.js` mais l'icon button ne l'implémente pas. Donc je pense qu'il faut remplacer par le `:disabled` natif

Reply:
Good catch